toxcore/TCP_server.c

```diff
@@ -222,6 +222,23 @@ static int add_accepted(TCP_Server *tcp_server, const Mono_Time *mono_time, cons
     return index;
 }
 
+static void wipe_priority_list(TCP_Secure_Connection *con)
```

Comment:
It'll need to be called something else. https://travis-ci.org/TokTok/c-toxcore/jobs/434939506#L1504